src/components/GameCard/PriceBox/PriceBox.tsx

```diff
@@ -1,37 +1,60 @@
 import React from "react";
+import classNames from "classnames";
 
+import { Promotion } from "../../../types/Game.ts";
+import { calculateDiscount } from "../../../utils/calculateDiscount.ts";
 import { TLabel } from "../../ui/TranslationLabel/TLabel.tsx";
+import CartButton from "../CartButton/CartButton.tsx";
 
 import "./PriceBox.css";
 
 interface PriceBoxProps {
   price: number;
-  oldPrice?: number;
+  promotion?: Promotion | null;
+  isOwned?: boolean;
+  onAddToCart?: () => void;
 }
 
-const PriceBox: React.FC<PriceBoxProps> = ({ price, oldPrice }) => {
+const PriceBox: React.FC<PriceBoxProps> = ({
```

Comment:
Il faut donc export ici
```suggestion
export const PriceBox: React.FC<PriceBoxProps> = ({
```